app/templates/addParticipants.html

```diff
@@ -0,0 +1,76 @@
+{% extends "base.html" %}
+
+{% block scripts %}
+{{super()}}
+  <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+  <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>

```

Comment:
we don't want to be including jquery UI and an old bootstrap here. I think this was old and we discussed it in the meeting.